README.md

```diff
@@ -1 +1,30 @@
-# 4-way-Traffic-Light-Prototype-with-Arduino
\ No newline at end of file
+# A 4-way-Traffic-Light-Prototype-with-Arduino
+
+This project aims at mimicking the process of controlling **a 4-way Traffic light system** using a Programmable Logical Controller
+The project provides a virtual environment for testing, as well as a miniature physical environment to show the output of the earlier conducted simulation in real time.
+
+
+## Table of Content
+1. Introduction
+2. Features of the project
+3. Procedure
+4. Contributors
+5. Access
+
+### Introduction
+As the population of the environment increases, the road users increases as well, thus, the need to control the order in which the road is used.
+Traffic light systems play a crucial role in controlling the flow of vehicles and pedestrians at intersections.
+A Programmable logical controller (PLC) is employed in place of human to control this flow by managing the timing and sequencing of traffic lights.
```

Comment:
We are not using a PLC, I think. We are just using Arduino, which is a microcontroller, to mimic what PLC could have executed.

Reply:
Would like that. Thank you